pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -854,3 +879,14 @@ func NewDomainWithSlirpInterface() *api.Domain {
 
 	return domain
 }
+
+func NewDomainWithMacvtapInterface(mavtapName string) *api.Domain {
+	domain := &api.Domain{}
+	domain.Spec.Devices.Interfaces = []api.Interface{{
+		Alias: &api.Alias{
+			Name: mavtapName,
+		},
+		Type: "ethernet",
```

Comment:
```
Model: &api.Model{
			Type: "virtio",
		},
```
is missing

Reply:
Done.

Reply:
where?

Reply:
Thanks for keeping me honest; must have been lost on a stash somewhere. Sorry for this.

(Now) Done (for real).